website/docs/components/tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,24 @@
+## Component API
+
+### TooltipButton
+
+text, placement, offset
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="text" @type="string">
+    Text string which will appear in the tooltip. May contain basic HTML tags for formatting text such as `strong` and `em` tags. Not intended for multi-paragraph text or other more complex content. May not contain interactive content such as links or buttons.
+  </C.Property>
+  <C.Property @name="placement" @type="enum" @values={{array "top" "top-start" "top-end" "right" "right-start" "right-end" "bottom" "bottom-start" "bottom-end" "left" "left-start" "left-end" }} @default="top">
+    Placement for the tooltip relative to the opener element. Will automatically shift position to remain visible when near the edges of the screen.
+  </C.Property>
+  <C.Property @name="offset" @type="array">
+    Takes two numbers. The first number controls “skidding” or the tooltip displacement from the center of the opener element. The second number controls the “distance” from the opener element.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Tooltip modifier/helper
+
+The helper offers the same options as the component See the “How to use” section for syntax examples.
```

Comment:
Good idea. I can add that.

Reply:
Oh wait, I think I misunderstood about the link. I was thinking you meant to add a link to Ember documentation on modifiers. (wasn't reading closely)

Do you mean moving the information related to the modifier into its own tab? Because I'm not sure there's enough that it makes sense to do that. Also personally, I find it useful to see the different syntax of the component vs. the modifier together on the same page for comparison.

Reply:
Sorry, my bad: I meant an anchor link to `/components/tooltip?tab=code#hds-tooltip-modifier`